.rubocop_todo.yml

```diff
@@ -0,0 +1,18 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config --exclude-limit 0`
+# on 2021-01-08 23:22:24 UTC using RuboCop version 1.8.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: Include.
+# Include: **/*.gemspec
+Gemspec/RequiredRubyVersion:
```

Comment:
Do we need this? It seems like you set it in the gemspec file

Reply:
If guess not https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version